src/um/ws2tcpip.rs

```diff
@@ -3,16 +3,345 @@
 // <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
 // All files in the project carrying such notice may not be copied, modified, or distributed
-// except according to those terms.
+// except according to those terms
 //! WinSock2 Extension for TCP/IP protocols
-
 use ctypes::c_int;
-use shared::minwindef::DWORD;
-use um::winsock2::LPWSAOVERLAPPED;
-
+use shared::guiddef::LPGUID;
+use shared::minwindef::{DWORD, INT, LPHANDLE, ULONG};
+use shared::mstcpip::{
+    SOCKET_PEER_TARGET_NAME, SOCKET_SECURITY_QUERY_INFO, SOCKET_SECURITY_QUERY_TEMPLATE,
+    SOCKET_SECURITY_SETTINGS,
+};
+use shared::winerror::{
+    WSAEAFNOSUPPORT, WSAEINVAL, WSAESOCKTNOSUPPORT, WSAHOST_NOT_FOUND, WSANO_RECOVERY,
+    WSATRY_AGAIN, WSATYPE_NOT_FOUND, WSA_IPSEC_NAME_POLICY_ERROR, WSA_SECURE_HOST_NOT_FOUND,
+};
+use shared::ws2def::{
+    ADDRINFOA, ADDRINFOEXA, ADDRINFOEXW, ADDRINFOW, PADDRINFOA, PADDRINFOEXA, PADDRINFOEXW,
+    PADDRINFOW, SOCKADDR, SOCKET_ADDRESS,
+};
+use shared::wtypesbase::LPBLOB;
+use um::minwinbase::LPOVERLAPPED;
+use um::winnt::{PCHAR, PCSTR, PCWSTR, PSTR, PVOID, PWCHAR, PWSTR, VOID};
+use um::winsock2::{
+    LPWSAOVERLAPPED, LPWSAOVERLAPPED_COMPLETION_ROUTINE, SOCKET, WSA_NOT_ENOUGH_MEMORY, timeval,
+};
+use vc::vcruntime::size_t;
+pub const UDP_NOCHECKSUM: c_int = 1;
+pub const UDP_CHECKSUM_COVERAGE: c_int = 20;
+pub const EAI_AGAIN: DWORD = WSATRY_AGAIN;
+pub const EAI_BADFLAGS: DWORD = WSAEINVAL;
+pub const EAI_FAIL: DWORD = WSANO_RECOVERY;
+pub const EAI_FAMILY: DWORD = WSAEAFNOSUPPORT;
+pub const EAI_MEMORY: DWORD = WSA_NOT_ENOUGH_MEMORY as DWORD;
```

Comment:
https://github.com/retep998/winapi-rs/blob/dev/CONTRIBUTING.md#constants

I accidentally forgot to commit and push those changes until just now...